src/OrbitCaptureClient/CaptureEventProcessorTest.cpp

```diff
@@ -479,21 +393,22 @@ TEST(CaptureEventProcessor, CanHandleInternedTracepointEvents) {
   EXPECT_EQ(actual_tracepoint_info.name(), tracepoint_intern->name());
   EXPECT_EQ(actual_tracepoint_event.pid(), tracepoint->pid());
   EXPECT_EQ(actual_tracepoint_event.tid(), tracepoint->tid());
-  EXPECT_EQ(actual_tracepoint_event.time(), tracepoint->time());
+  EXPECT_EQ(actual_tracepoint_event.time(), tracepoint->timestamp_ns());
   EXPECT_EQ(actual_tracepoint_event.cpu(), tracepoint->cpu());
 }
 
 constexpr int32_t kGpuPid = 1;
 constexpr int32_t kGpuTid = 2;
```

Comment:
Now that they're out of the anonymous namespace, I'd prefer these to be explicitly static.

Reply:
This is very minor but still open (yes, GitHub makes it hard to find).

Reply:
done.